File: smart_app_project/src/main/java/com/example/demo/domain/SubmissionRequest.java
package com.example.demo.domain;

public class SubmissionRequest {

    private Long problemId;
    private Long userId;
    private String userAnswer;
    private int checkCount;

    // 🔥 [신규 추가] 공부 시간 (초 단위)
    private int studyTime;

    // --- Getter & Setter ---
    public Long getProblemId() { return problemId; }
    public void setProblemId(Long problemId) { this.problemId = problemId; }

    public Long getUserId() { return userId; }
    public void setUserId(Long userId) { this.userId = userId; }

    public String getUserAnswer() { return userAnswer; }
    public void setUserAnswer(String userAnswer) { this.userAnswer = userAnswer; }

    public int getCheckCount() { return checkCount; }
    public void setCheckCount(int checkCount) { this.checkCount = checkCount; }

    // 🔥 추가된 필드의 Getter/Setter
    public int getStudyTime() { return studyTime; }
    public void setStudyTime(int studyTime) { this.studyTime = studyTime; }
}

File: MyQuizApp-main/app/src/main/java/com/example/myapplication/data/model/SubmissionRequest.kt
package com.example.myapplication.data.model

data class SubmissionRequest(
    val problemId: Long,
    val userId: Long,
    val userAnswer: String,
    val checkCount: Int,

    // 🔥 [신규 추가] 기본값 0 (혹시 모를 에러 방지)
    val studyTime: Int = 0
)